### ISSUE TITLE
PoS Window Upgrade: Not clear where in the 2016 block window we are

### ISSUE THREAD

**Issue by NickEngland** [decred/hardforkdemo#83]
In the PoS Window Blowup, it's not clear where in the 2016 window we are currently. This would be useful to help see if we are on target for the 75% or not.

Perhaps on the stacked chart on the right, you could change the Y Axis to the 10080 to get a feel for the number left?

On the non-blowup, the bar graph filling up with votes with the 75% marked, perhaps votes which aren't V4 should show filling in from the right, so you can see visually if there is enough votes left in the window to make the 75% or not?

Also, isn't 5*2016=10080 votes, not 10012 votes?

**Comment by alexlyp** [comment]
I just recently updated the site, so take another look.

For the difference between 10080 and 10012, while that's true that is the total number of vote in the interval.  If any votes were missed, they are basically abstains.  Or in a different way to say it:  a vote version must be 75% of votes issued (not total available votes).  (eg blocks that have 3/5 or 4/5 votes)

**Comment by alexlyp** [comment]
I agree that some work can be done to make it more clear as far as stake interval progress

**Comment by NickEngland** [comment]
Would it be possible to show where in the static window you are with a line, similar to the % votes line?

So you would see:
Window
`<----------------><                >`
Votes
`<------------><                     >`

To visually see if we are behind or ahead of the needed votes?

Or you could combine them in one bar, so the non-V4 votes fill in right-to-left, and the v4 Left-to-right. When they meet the window is complete. That way you could see if we are on track to hit, or couldn’t possibly hit the 75% target?
Combined:
`<--------------><             |  ><+++++>`

Since you have the V4 in green, and V3 in red boxes, might make sense to make the progress bar of V4 votes green too?
